docs/tutorials.md

```diff
@@ -0,0 +1,59 @@
+# Tutorials
+
+In addition to the basic getting started guides, we offer tutorials that aim to
+show how Material for MkDocs can be used in different use cases. In contrast to
+the getting started guides or the reference documentation, the tutorials show
+the breadth of functionality available with Material for MkDocs but also within
+the wider MkDocs ecosystem.
+
+* [Blogs](tutorials/blogs.md)
+
+## Tutorial Structure
+
+The tutorials guide you through worked examples, so following them you should
+gain not only and understanding of how Material for MkDocs can be used but also
+a template for your own projects. For convenience, these templates are also
+available as [template repositories on GitHub].
+
+<!-- TODO -->
+[template repositories on GitHub]: templates.md
+
+Each tutorial is a single page, so use the Table of Contents navigation to move
+between numbered steps. They start by mentioning prerequisites and giving an
+overview of the content as well as describing the outcomes.
+
+### Instructions
+
+Tutorials contain a mix of explanation, demonstrations, and instructions that you
+should follow as you work your way through the individual steps. Those instructions
+are shown in the form of admonitions to separate them from the rest of the
+tutorial:
+
+!!! example "Instructions to follow"
+
+    Whenever you see a box  like this you know it is time to get hands on and
+    apply material just covered as well as reflect on how you would it in your
+    work work.
+
+### Tips and warnings
+
+Likewise, helpful tips and warnings appear in boxes of their own:
+
+!!! tip "A hot tip!"
+
+    Tips look like this...
+
+!!! warning "Warning!"
+
+    ...and warnings like this!
+
+## Feeback wanted
+
+The tutorials are a recent addition to our documentation and we are still
+working out what shape they should have in the end. Please contact us if you
+want to provide feedback. <!--- TODOD: how? -->
```

Comment:
We should create a discussion asking for feedback on this tutorial, pin it, and link to that.